hols/oci-basic-setup/README.md

```diff
@@ -0,0 +1,203 @@
+       ```
+    2. Test the Hello world request.
+       ```shell
+       curl http://$ENDPOINT_IP:8080/greet
+       ```
+       results in:
+       ```shell
+       {"message":"Hello World!","date":[2025,4,29]}
+       ```
+    3. Test Hello to a name, i.e. to `Joe`.
+       ```shell
+       curl http://$ENDPOINT_IP:8080/greet/Joe
+       ```
+       results in:
+       ```shell
+       {"message":"Hello Joe!","date":[2025,4,29]}
+       ```
+    4. Replace Hello with another greeting word, i.e. `Hola`.
+       ```shell
+       curl -X PUT -H "Content-Type: application/json" -d '{"greeting" : "Hola"}' http://$ENDPOINT_IP:8080/greet/greeting 
+       curl http://$ENDPOINT_IP:8080/greet
+       ```
+       results in:
+       ```shell
+       {"message":"Hola World!","date":[2025,4,29]}
+       ```
+2. Validate that the Helidon Metrics are pushed to the OCI Monitoring Service using the OCI metric integration that was added in the Helidon application:
+   1. From the OCI Console, go to `Observability & Management` -> `Metrics Explorer (under Monitoring)`.
+   2. On `Query 1`, choose the `Compartment` value with the format of `devops-compartment-helidon-demo-<4 char random value>`.
+   3. Select `helidon_metrics` under `Metric namespace`.
+   4. Select `requests.count_counter` under `Metric Name`.
+   5. Above the empty graph, you can choose values for `Start time/End Time` or choose the time duration under `Quick Selects`.
+   6. Click `Update Chart` at the bottom of `Query 1` to show all the metric data for the `request count`. You can also toggle to enable `Show Data Table` on the right upper portion of the graph to show the list of data for the chosen metric.
+   7. You can also explore other Metrics by going back to step 4 and choosing a new value. 
+3. Validate OCI Logging SDK integration that was added in the Helidon application. This will push log messages to the OCI Logging Service:
+   1. From the OCI Console, go to `Observability & Management` -> `Logs (under Logging)`.
+   2. Change `Compartment` with a value that has the format of `devops-compartment-helidon-demo-<4 char random value>`.
+   3. Under `Filters`, change `Log Group` to `app-log-group-helidon-demo`.
+   4. Choose and click on `app-log-helidon-demo` from the Logs table.
+   5. Choose `Filter by time` value within the scope of your last request. For example, you can choose `Today` to see all requests that were made today.
+   6. The `Explore Log` display should output graphs of the logging activity. Below the graphs, it will also show a list of the logs that have been captured.
+
+### Cleanup
+When the environment is no longer needed, all the OCI resources can be cleaned up by following these steps:
+1. Go back to oci-basic-setup local repository directory.
+   ```shell
+   cd ~/helidon-labs/hols/oci-basic-setup
+   ```  
+2. Execute the `destroy` option from terraform.
+   ```shell
+   terraform destroy -auto-approve
+   ```   
```

Comment:
Consider adding a troubleshooting section as we discussed

Reply:
Updated